recipe/meta.yaml

```diff
@@ -1,39 +1,56 @@
 {% set name = "hyperframe" %}
-{% set version = "6.0.1" %}
-{% set sha256 = "ae510046231dc8e9ecb1a6586f63d2347bf4c8905914aa84ba585ae85f28a914" %}
+{% set version = "6.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: f630908a00854a7adeabd6382b43923a4c4cd4b821fcb527e6ab9e15382a3b08
 
 build:
+  skip: true  # [py<39]
   number: 0
-  noarch: python
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
 
 requirements:
   host:
-    - python >=3.6
-    - setuptools
+    - python
+    - pip
+    - setuptools >=75.6.0
+    - wheel >=0.45.0,<1
```

Comment:
```suggestion
```
`wheel` is redundant if setuptools >=75.6.0. Since setuptools 70.1.0 the code from the wheel project is vendored into setuptools directly, see:

-  https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use
-  https://setuptools.pypa.io/en/latest/history.html#v70-1-0